docs/_includes/tests/breadcrumb.html

```diff
@@ -33,10 +62,27 @@ <h2>Buttons, Current Page</h2>
         <nav aria-labelledby="breadcrumbs-heading" class="breadcrumb" role="navigation">
             <h2 id="breadcrumbs-heading" class="clipped">You are here</h2>
             <ul>
-                <li><button>ebay</button></li>
-                <li><button>Cell Phones, Smart Watches &amp; Accessories</button></li>
-                <li><button>Smart Watch Accessories</button></li>
-                <li><button aria-current="location">Smart Watch Bands</button></li>
+                <li>
+                    <button>ebay</button>
+                    <svg class="icon icon--breadcrumb" focusable="false" height="8" width="8" aria-hidden="true">
```

Comment:
Only thing I would comment on this is, is the icon part of the previous element or next element?
I definitely think it's fine having `svg` for each one, but I think grouping it with each element seems strange and not right. I would like to see something like: 
```
<ul>
   <li> Item </li>
   <svg>
   <li> Item 2 </li>
</ul>
``` 
but I know that's not proper. Maybe put the `<svg>` in a `<li>`?

Reply:
Yep, we can't put the sag tag between list items.

Originally I looked at doing this:

```
<ul>
   <li> Item </li>
   <li role="presentation"><svg></svg></li>
   <li>Item 2</li>
</ul>
```

But the screen readers were still including the breadcrumb in the list count (list count should ideally say "two" in the example above, not "three".

I might try this though:

```
<ul>
   <li> Item </li>
   <li aria-hidden="true"><svg></svg></li>
   <li>Item 2</li>
</ul>
```

Reply:
Yes, I think the last one is best, if it works for screen readers maybe we should go with that approach. 
If not, then we can do it this way

Reply:
I've played around with it, and only the way to guarantee that the list item count is announced as 4, rather than 7, is to use the current markup. I'm going to keep it as is for now, but there's plenty of time to come back and revisit before final release.